docs/storage.md

```diff
@@ -160,9 +165,24 @@ For testing:
 
 `Storage Object Admin` for ability to create and delete temporary buckets.
 
-
 ## Other minio supported S3 object storages
 
 Minio client used for AWS S3 can be potentially configured against other S3-compatible object storages.
 
 <TBD>
+
+## Azure Configuration
+
+To use Azure Storage as Thanos object store, you need to precreate storage account from Azure portal or using Azure CLI. Follow the instructions from Azure Storage Documentation: [https://docs.microsoft.com/en-us/azure/storage/common/storage-quickstart-create-account](https://docs.microsoft.com/en-us/azure/storage/common/storage-quickstart-create-account?tabs=portal)
+
+To configure Azure Storage account as an object store you need to provide a path to Azure storage config file in flag `--objstore.config-file`.
+
+Config file format is the following:
+
+```yaml
+type: AZURE
+config:
+    storage-account: <Name of Azure Storage Account>
```

Comment:
can we aim for camelCase style for fields? like Kubernetes: https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/